src/util/csv.ts

```diff
@@ -1,13 +1,40 @@
 import Papa from "papaparse";
 import { Address } from "../types/records";
 
-export const addressesToCsvString = (addresses: Address[]): string => Papa.unparse(
-  addresses.map((address) => ({ key: address.key, val: address.val }))
-);
+export const getAddressKey = (address): string => {
+  if (address.Name) return address.Name;
+  if (address.name) return address.name;
+  if (address.Key) return address.Key;
+  if (address.key) return address.key;
+  return "";
+};
+
+export const getAddressVal = (address): string => {
+  if (address.Address) return address.Address;
+  if (address.address) return address.address;
+  if (address.Val) return address.Val;
+  if (address.val) return address.val;
+  return "";
+};
+
+export const addressesToCsvString = (addresses: any[]): string => {
+  const addressList = addresses.map((address) => {
+    return {
+      name: address.key,
+      address: address.val,
```

Comment:
So is this search now not case sensitive? I'm confused

Reply:
This is taking the addresses from the Lattice, which are always {key, val} and turning them into {name, address} so users see "name" and "address" as the headers in their CSV file.

Reply:
Right, I was referring to the capitalized keys that show up in this PR

Reply:
ohhh--yeah, it's just making it a little less case-sensitive. No other functionality in the app should change outside of the CSV headers. The user was having issues with capitalization with the header names.